sources/procfs.go

```diff
@@ -528,8 +528,11 @@ func getJobStatsIOMetrics(jobBlock string, jobID string, promName string, helpTe
 
 func getJobNum(jobBlock string) (jobID string, err error) {
 	jobID = regexCaptureString("job_id: .*", jobBlock)
-	jobID = regexCaptureString("[0-9]*\\.[0-9]+|[0-9]+", jobID)
-	return strings.Trim(jobID, " "), nil
+	jobIDlist := regexCaptureNumbers(jobID)
+	if len(jobIDlist) < 1 {
```

Comment:
FWIW, I realize we'd need to change several other bits of code if we decide to check for '!= 1' rather than '< 1'. Just raising it because I see it here, should have asked earlier.